src/main.py

```diff
@@ -59,11 +58,10 @@ async def main() -> None:
     config = RuntimeConfig.load_from_yaml(args.config)
 
     tasks = [job.run() for job in config.jobs]
-    for job, completed_task in zip(
+    for _job, completed_task in zip(
         config.jobs, asyncio.as_completed(tasks), strict=False
     ):
         await completed_task
-        log.info("Job completed: %s", job)
```

Comment:
If we aren't using job here anymore, then we don't need the zip any more... do we?